hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminClient.java

```diff
@@ -76,7 +82,6 @@ public RSGroupInfo getRSGroupInfo(String groupName) throws IOException {
     }
   }
 
-  @Override
   public RSGroupInfo getRSGroupInfoOfTable(TableName tableName) throws IOException {
```

Comment:
This method can be removed?

Reply:
I think we will reference this method in hbase shell so do not want to change it in this patch...

Let's do it in another issue. And in general, if we do not want to completely remove the shell command, we'd better keep this method for a least a whole major version?

Reply:
We can change the impl of "get_table_rsgroup" cmd? Then there will no internal usage for this.